config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml

```diff
@@ -11,106 +11,114 @@ spec:
     kind: VolumeSnapshot
     plural: volumesnapshots
   scope: ""
+  validation:
+    openAPIV3Schema:
+      description: VolumeSnapshot is a user's request for taking a point in time snapshot
```

Comment:
How about:

"VolumeSnapshot is a user's request for taking a point-in-time snapshot of a PersistentVolumeClaim. Upon successful creation of the snapshot by the storage provider..."